rpcd/playbooks/roles/rpc_maas/tasks/raxmon_agent_install.yml

```diff
@@ -38,6 +38,11 @@
     group: "root"
     mode: "0600"
 
+- name: Verify raxmon-entities-list and MaaS API authentication does not fail
+  shell: "raxmon-entities-list"
```

Comment:
They are done with shell because they require the use of pipes. [Best practice](http://docs.ansible.com/ansible/shell_module.html#notes) is to use command where possible.

As an aside they are tasks not plays. Plays are found in a playbook. Tasks are found in a tasks file.